lib/action_cable/connection/callbacks.rb

```diff
@@ -0,0 +1,61 @@
+module ActionCable
+  module Connection
+    # = Action Cable Callbacks
+    #
+    # Action Cable provides hooks during the life cycle of a connection.
+    # Callbacks allow you to trigger logic during the life cycle of a
+    # connection. Available callbacks are:
+    #
+    # * <tt>after_connect</tt>
+    # * <tt>after_disconnect</tt>
+    #
+    module Callbacks
+      extend  ActiveSupport::Concern
+      include ActiveSupport::Callbacks
+
+      included do
+        define_callbacks :connect, scope: :name
+        define_callbacks :disconnect, scope: :name
+      end
+
+      module ClassMethods
+        # Defines a callback that will get called right after the
+        # connection is established.
+        #
+        #   module ApplicationCable
+        #     class Connection < ActionCable::Connection::Base
+        #       identified_by :current_user
+        #       after_connect :prepare_connection
```

Comment:
So after we've connected, we prepare the connection? It sounds like `prepare_connection` doesn't make sense as an `after_connect` callback.